### ISSUE TITLE
Postprocessing / denormalization

### ISSUE THREAD

**Issue by frauzufall** [bioimage-io/configuration#39]
Hi all,

maybe I'm missing something obvious, but where can I specify how the result is denormalized? Since one can now add preprocessing to any input node, should each output node not also have an optional postprocessing parameter? Or is this also called `preprocessing` for output nodes?

Best, Debo

**Comment by fjug** [comment]
We were discussing in all brevity about such options but then concluded its not needed. (It was also after quite some hours of intense discussion...)
Now that you point, you actually have a point. For virtually all image restoration applications we indeed want to bring the outputs back in the original image range...
I think we should discuss this in tomorrows meeting when we are all present.

**Comment by FynnBe** [comment]
I think it would be great to have an analog `postprocessing` for the outputs. 👍

**Comment by oeway** [comment]
Hi, can't you just add the denormalization op into the network itself when you export it?

**Comment by frauzufall** [comment]
I'm just wrapping existing networks, I'm not designing them. You could also add normalization to the network, no? If there's a way to specify normalization here, it just seems weird that there is no way to specify denormalization.

**Comment by FynnBe** [comment]
Decided in today's bioimage.io meeting:
- design `postprocessing` analog to `preproessing`

for valid postprocessings we start with:
- `sigmoid`, no kwargs

- `scale_mean_variance` with `kwargs`:
  - `mode`: 'per_dataset' / 'per_sample'
  - `reference_input`: name of input tensor to derive mean and variance from
  - (note: if you want to use fixed mean, std, use `scale_linear`)

- `scale_min_max` with `kwargs`:
  - `reference_input`: name of input tensor to use for statistics
  - `mode`: (fixed/per_dataset/per_sample)
  - `min_percentile`: (if `mode` is not 'fixed') default 0
  - `max_percentile`: (if `mode` is not 'fixed') default 100 (or 1.0?)
  - `min`: (if `mode` is 'fixed')
  - `max`: (if `mode` is 'fixed')

-  `scale_linear` with `kwargs`:
    - 'gain': default 1.0
    - 'offset': default 0.0

```yaml

outputs
 - name: output_logits
   drange: ...
   ...
   postprocessing: {name: sigmoid, kwargs: {}}
```

todos:
 - [ ] PR

**Comment by constantinpape** [comment]
We have added the `postprocessing` description in #43 now.
